src/main/java/org/galatea/starter/service/IexClientHistorical.java

```diff
@@ -0,0 +1,51 @@
+package org.galatea.starter.service;
+
+import java.util.List;
+import org.galatea.starter.domain.IexHistoricalPrice;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+/**
+ * A Feign Declarative REST Client to access endpoints from the Free and Open IEX API to get market
+ * data. This client uses a different url to the other IexClient.
+ */
+@FeignClient(name = "IEXHistorical", url = "${spring.rest.iexHistoricalPath}")
+public interface IexClientHistorical {
```

Comment:
So you've called this `IexClientHistorical` because right now it you've only defined endpoints for Historical Prices but the `https://cloud.iexapis.com/` endpoint supports a variety of endpoints (ex: Fund Ownership endpoints `GET /stock/{symbol}/fund-ownership`). If the next thing you were asked to do was to expose Fund Ownership in this FUSE project, one of the first things you would need to change would be the name of this interface. I'd probably just have it be a more generic name to start out with to represent it covers any endpoints under `https://cloud.iexapis.com/`